samples/express.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2018, Google, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START logging_bunyan_express]
+// Imports the Google Cloud client library for Bunyan.
+const lb = require('@google-cloud/logging-bunyan');
+
+// Import express module and create an http server.
+const express = require('express');
+
+async function startServer() {
+  const {logger, mw} = await lb.express.middleware();
```

Comment:
Lack of top-level await is annoying.

It is possible to make `lb.express.middleware` non-async, but async is The Right Way to go. `middleware` does the necessary initialization to setup the Stackdriver Logging transport, and errors, if any should be thrown/caught at this point by the users. Making it non-async would defer the error to be thrown in the request handlers which would be The Wrong Way.